dotCMS/src/main/java/com/dotmarketing/image/filter/CropImageFilter.java

```diff
@@ -64,4 +112,27 @@ public File runFilter(File file,  Map<String, String[]> parameters) {
 		return resultFile;
 	}
 
+
+    protected Optional<Point> calcFocalPoint(BufferedImage src, Map<String, String[]> parameters) {
+        Dimension current = new Dimension(src.getWidth(), src.getHeight());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'current' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)